python/rikai/contrib/torch/transforms/yolov5.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import Any, Callable, Dict
+
+
+__all__ = ["post_processing", "OUTPUT_SCHEMA"]
```

Comment:
don't need preprocesing?

Reply:
yeah, the yolo interface can take uri numpy pil etc

Reply:
nice